addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/EnvFileScanRuleUnitTest.java

```diff
@@ -87,7 +106,7 @@ public void shouldAlertWhenRequestIsUnauthorizedAtLowThreshold() throws Exceptio
                         newFixedLengthResponse(
                                 Response.Status.UNAUTHORIZED,
                                 NanoHTTPD.MIME_HTML,
-                                IRRELEVANT_BODY)));
+                                RELEVANT_BODY)));
```

Comment:
Please revert, this indicates that neither the Content-Type nor the content is relevant for the outcome of the test.